velox/common/memory/MemoryUsageTracker.h

```diff
@@ -60,6 +60,22 @@ constexpr int64_t kMaxMemory = std::numeric_limits<int64_t>::max();
 class MemoryUsageTracker
     : public std::enable_shared_from_this<MemoryUsageTracker> {
  public:
+  /// The memory usage tracker's internal stats.
+  struct Stats {
+    uint64_t peakBytes{0};
+    uint64_t cumulativeBytes{0};
+    uint64_t numAllocs{0};
+    uint64_t numFrees{0};
+    uint64_t numReserves{0};
+    uint64_t numReleases{0};
+    uint64_t numCollisions{0};
```

Comment:
Some of these counters might be straightforward, but numCollisions is probably not. Would you document it? Also, consider documenting cumulativeBytes as it may not be obvious as well.

Reply:
Added. Thanks!